src/LookingGlassControls.ts

```diff
@@ -22,32 +23,64 @@ export function initLookingGlassControlGUI(lkgCanvas: HTMLCanvasElement) {
 	c.style.borderRadius = "10px"
 	c.style.right = "15px"
 	c.style.bottom = "15px"
+	c.style.flex = "row"
 
 	const title = document.createElement("div")
 	c.appendChild(title)
 	title.style.width = "100%"
 	title.style.textAlign = "center"
 	title.style.fontWeight = "bold"
-	title.innerText = "Looking Glass Controls "
+	title.style.marginBottom = "8px"
```

Comment:
Consider removing commented out code to a separate PR, this functionality was a test for using the mediaStream functionality of browsers, but needs more testing before it should be merged into main.

Reply:
resolved: https://github.com/Looking-Glass/looking-glass-webxr/pull/31/commits/5a1738791e3cc878f3865d46c4bc4605f2a6f770